alg/sha256_arm.c

```diff
@@ -91,14 +72,14 @@ SHA256_Transform_arm(uint32_t state[static restrict 8],
 	uint32x4_t S0_orig, S1_orig;
 	int i;
 
+	(void)W; /* UNUSED */
```

Comment:
Now that these are unused, is there any reason not to drop them like we do for the x86 shani version?

Reply:
I considered it, but slightly erred on the side of simplicity; particularly because it would require a `sha256_Transform_arm_with_W_S` shim function in `sha256.c`.  But it was only a 55% to 45% "erring", so I'm quite content to add that.